shimmer/develop/what-is-shimmer/clients.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Become a Node Operator
+description: "You can use the IOTA client libraries to handle communications between your node and
+your Layer 1 network of choice."
+image: /img/shimmer-wiki.png
+---
+
+# IOTA Client Libraries
+
+:::note tl;dr;
+
+You can use the IOTA client libraries to handle communications between your node and
+your [Layer 1 network](layer-1-landscape.md) of choice.
+
+The IOTA foundation recommends using [wallet.rs](#walletrs) for value transactions.
+
+:::
+
+:::tip Free and Open-Sourced
+
+All the libraries mentioned in this section are free, open-sourced, and maintained by both
+the [IOTA Foundation](https://www.iota.org/) and
+the [IOTA Community](https://wiki.iota.org/shimmer/community/the-community/how-to-support/).
+
+:::
+
+## Simple by Default, as Complex as You May Need
+
+IOTA Client libraries handle the communications between your software and
+the Shimmer network. They are designed to simplify how you connect and interact with nodes in the network by providing
+high-level helper functions which combine multiple atomic REST-API commands under the hood and use sensible defaults.
+However, you can still customize your application by overriding the default parameters or combining atomic REST-API
+commands to suit your needs.
+
+## High-level Overview
+
+|                                    | [wallet.rs](#walletrs) | [iota.rs](#iotars) | [iota.js](iotajs) |
```

Comment:
Not sure about this table. 
Do we recommend iota.rs for value transactions? And iota.js we don't?

Reply:
My bad, mixed up there, corrected